app/models/export/submissions/row.rb

```diff
@@ -0,0 +1,97 @@
+module Export
+  class Submissions
+    class Row
+      MISSING = '#MISSING'.freeze # fields that are needed for MVP that we don't have yet
+
+      attr_reader :submission
+
+      def initialize(submission)
+        @submission = submission
+      end
+
+      def row_values
+        [
+          submission.task_id,
+          submission.id,
+          status,
+          submission_type,
+          submission_file_type,
+          order_entry_count,
+          order_value,
+          invoice_entry_count,
+          invoice_value,
+          management_charge_value,
+          management_charge_rate,
+          created_date,
+          created_by,
+          supplier_approved_date,
+          supplier_approved_by,
+          finance_export_date,
+          po_number
+        ]
+      end
+
+      def status
+        {
+          'completed'         => 'supplier_accepted',
+          'validation_failed' => 'validation_failed'
+        }.fetch(submission.aasm_state)
+      end
+
+      def submission_type
+        (invoice_entry_count + order_entry_count).zero? ? 'no_business' : 'file'
+      end
+
+      def submission_file_type
+        File.extname(submission._first_filename).downcase[1..-1] if submission._first_filename
+      end
+
+      def management_charge_value
+        format('%0.2f', submission.management_charge / 100.0) if submission.management_charge
+      end
+
+      def management_charge_rate
+        format('%0.3f', Framework::TMP_FIXED_CHARGE_RATE / 100)
```

Comment:
although not a problem here where we have a fixed value of 1.5, there are frameworks that have more decimal places in their management charge (e.g. GCloud with 0.75%) so I think for safety we should tweak this as the current format string would round that to `0.008`.